src/Components/Header/Header.css

```diff
@@ -0,0 +1,6 @@
+h1 {
+  font-family: 'Playfair Display SC', serif;
```

Comment:
just want to say, I only added one font for an accent style and I am not at all committed to it, so if you like any fonts I am down for it!